Northstar.Custom/mod/scripts/vscripts/sh_custom_air_accel.gnut

```diff
@@ -29,4 +30,12 @@ void function ApplyCustomPlayerAirAccelOnLoadoutChange( entity player, PilotLoad
 {
 	player.kv.airAcceleration = GetCurrentPlaylistVarInt( "custom_air_accel_pilot", int( player.GetPlayerSettingsField( "airAcceleration" ) ) )
 }
+void function ApplyDefaultPlayerAirAceelOnDisconnect( entity player )
+{
+	int defaultairaccel = 500
+    if ( !multiplayer() )
+		return 0
```

Comment:
the issue here is that playlist vars persist across multiple servers, not that air accel specifically persists, we probably shouldn't add hack fixes to fix individual playlist vars, and instead should have a more general solution for playlist vars persisting like this if it's an issue

Reply:
It could totally be something else as it only  happened on a few servers so it seemed kinda finicky
If so just close this and forget it ever happened